api/catalog/api/constants/field_order.py

```diff
@@ -0,0 +1,46 @@
+"""
+This file hardcodes the logical order for serializer fields to be defined in so that
+their output feels suitable for human consumption.
+"""
+
+json_fields = [
+    "id",
+    "title",
+    "foreign_landing_url",
+    "url",
+    "creator",
+    "creator_url",
+    "license",
+    "license_version",
+    "license_url",
+    "provider",
+    "source",
+    "category",
+    "genres",  # audio-only
+    "filesize",
+    "filetype",
+    "tags",
+    "alt_files",  # audio-only
+    "attribution",
+    # Hits
+    "fields_matched",
+    "mature",
+    # Image specific
+    "height",
+    "width",
+    # Audio specific
+    "audio_set",
+    "duration",
+    "bit_rate",
+    "sample_rate",
+    # Hyperlinks
+    "thumbnail",
+    "detail_url",
+    "related_url",
+    "waveform",  # audio-only
+    # Add-ons
+    "peaks",  # audio-only
+]
+
+field_pos_map: dict[str, int] = {field: idx for idx, field in enumerate(json_fields)}
```

Comment:
Can we use the full name? does this variable mean `field_position_map`?

Reply:
Yeah it maps each field to its sorted position. Expanded the name in 3d68558.